app/gui/src/dashboard/hooks/backendBatchedHooks.ts

```diff
@@ -188,14 +190,55 @@ export function moveAssetsMutationOptions(backend: Backend) {
     mutationFn: async ([ids, parentId]: [ids: readonly AssetId[], parentId: DirectoryId]) => {
       const results = await Promise.allSettled(
         ids.map((id) =>
-          backend.updateAsset(id, { description: null, parentDirectoryId: parentId }, '(unknown)'),
+          backend
+            .updateAsset(
+              id,
+              { description: null, parentDirectoryId: parentId, title: null },
+              '(unknown)',
+            )
+            .catch((error) => {
+              if (error instanceof DuplicateAssetError) {
+                return { id, error }
+              }
+              throw error
+            }),
         ),
       )
 
+      const duplicateErrors = results
+        .filter((result) => result.status === 'fulfilled')
```

Comment:
consider flatMap instead of filter + map + filter

Reply:
I believe the first filter makes sense, but the map+filter can be replaced with flatMap. However, I’m not sure what improvement it would bring. We’ll still need to perform a double pass anyway.

Reply:
just a tradeoff on how it reads, i guess. instead of multiple chained methods where you have to read both the method name *and* the callback, it would just be one (more complex) function. plus the function below *doesn't* have the filter on status which makes it inconsistent